vernac/library.ml

```diff
@@ -103,25 +103,21 @@ type library_summary = {
   libsum_digests : Safe_typing.vodigest;
 }
 
-module LibraryOrdered = DirPath
-module LibraryMap = Map.Make(LibraryOrdered)
-module LibraryFilenameMap = Map.Make(LibraryOrdered)
```

Comment:
This kind of redeclaration has a semantic intent, I am not sure it should be removed.

Reply:
@ppedrot: I was actually about to tell something similar, that it has a "documentation" intent. So, maybe, a compromise would indeed be to keep this one.

Reply:
Functors are applicative so while they may be differentiated from DPmap by the mli these 1 are not different from each other though

Reply:
I did indeed wonder what to do about these, but in the end I found overall code clearer without the aliases, they don't seem to bring much to me (on the contrary they do obscure the code a bit), but no problem in keeping them if you find them useful.